src/pdm/cli/options.py

```diff
@@ -59,7 +59,7 @@ class ArgumentGroup(Option):
 
     def __init__(
         self,
-        name: str = None,
+        name: str,
```

Comment:
It seems [`ArgumentGround` is never instantiated without a name](https://github.com/pdm-project/pdm/search?q=ArgumentGroup) so my opinion is it would be fine just requiring it (making it non-optional).

Reply:
Ok